client/app/util/ProxyFactory.js

```diff
@@ -9,6 +9,8 @@ class ProxyFactory {
             target[prop].apply(target, arguments);
             hook(target);
           }
+        } else {
+          return target[prop];
```

Comment:
> O cangaceiro descobriu o motivo de sua cartucheira falhar alguns disparos.
The policy is:
when the prop is in the list, it must trigger the hook. Otherwise just call the target method without triggering it.